core-service/src/main/java/org/fundacionjala/sevenwonders/beans/AuthService.java

```diff
@@ -29,12 +27,23 @@ public AuthService(){
 
     public PlayerModel login(PlayerModel playerModel){
         PlayerModel player = new PlayerModel();
-        player.setId(autoIncrementId);
-        player.setUserName(playerModel.getUserName());
-        player.setToken(UUID.randomUUID().toString());
-        players.put(autoIncrementId,player);
+        if (playerModel.getUserName() != null && !playerModel.getUserName().isEmpty()) {
```

Comment:
I think it is better to validate at the beginning if the player is invalid, in that case throw an exception